sys/fmt/fmt.c

```diff
@@ -508,22 +509,17 @@ uint32_t scn_u32_hex(const char *str, size_t n)
 
 void print(const char *s, size_t n)
 {
-    if (IS_USED(MODULE_STDIO_NATIVE)) {
+    if (IS_USED(MODULE_STDIO_NATIVE) || IS_ACTIVE(LINUX)) {
```

Comment:
```suggestion
    if (IS_USED(MODULE_STDIO_NATIVE) || !defined(RIOT_VERSION)) {
```

I think this should also work